AmplifyPlugins/Storage/AWSS3StoragePlugin/Dependency/AWSS3Adapter.swift

```diff
@@ -8,24 +8,34 @@
 import Foundation
 import AWSS3
 
-/// The class confirming to AWSS3Behavior which uses an instance of the AWSS3 to perform its methods. This class acts
-/// as a wrapper to expose AWSS3 functionality through an instance over a singleton, and allows for mocking in unit
-/// tests. The methods contain no other logic other than calling the same method using the AWSS3 instance.
+/// The class confirming to AWSS3Behavior which uses an instance of the AWSS3 to perform its methods.
+/// This class acts as a wrapper to expose AWSS3 functionality through an instance over a singleton,
+/// and allows for mocking in unit tests. The methods contain no other logic other than calling the
+/// same method using the AWSS3 instance.
 class AWSS3Adapter: AWSS3Behavior {
     let awsS3: AWSS3
 
     init(_ awsS3: AWSS3) {
         self.awsS3 = awsS3
     }
 
+
```

Comment:
nit: extra whitespace. I think swiftlint would flag this, so make sure you're building locally before submitting the PR

```suggestion
```